### DISCUSSION TITLE
Predictive Weather Utilizing AI

### DISCUSSION THREAD

**Discussion by ghost** [Akylas/oss-weather#139] [Ideas]
With the abundance of AI models nowadays, it would be cool to see predictive weather that utilizes one of these models. 

Similar companies - https://www.rainbow.ai/business#about

**Comment by 0-5-0** [comment]
If you can find something that has a free API definitely share but I don't think any paid APIs would work well for this project.

I see GraphCast was recently added to the Open-Meteo models (then added to OSS Weather in the latest version) if that would cover your use case.

OSS Weather release note:
https://github.com/Akylas/oss-weather/releases/tag/android%2Fgithub%2F1.12.7%2F109

Open-Meteo GraphCast addition:
https://github.com/open-meteo/open-meteo/issues/701

Details about the weather model:
https://registry.opendata.aws/noaa-nws-graphcastgfs-pds/
